pkgs/by-name/pe/petsc/package.nix

```diff
@@ -174,10 +174,18 @@ stdenv.mkDerivation rec {
   # the library is installed and available.
   doInstallCheck = true;
   installCheckTarget = "check_install";
-  nativeInstallCheckInputs = [ mpiCheckPhaseHook ];
+  nativeInstallCheckInputs =
+    [
+      mpiCheckPhaseHook
+    ]
+    ++ lib.optionals pythonSupport [
+      python3Packages.pythonImportsCheckHook
+    ];
+
+  pythonImportsCheck = [ "petsc4py" ];
 
   passthru = {
-    inherit mpiSupport;
+    inherit mpiSupport pythonSupport;
```

Comment:
Same nit here - could be separated to a different commit.

Reply:
> Same nit here - could be separated to a different commit.

This nit still applies :) If you won't mind grabbing that in the next iteration it'd be nice :).